pick.c

```diff
@@ -851,7 +885,8 @@ print_choices(size_t offset, size_t selection)
 
 		if (i - offset < choices_lines)
 			print_line(choice->string, choice->length,
-			    i == selection, choice->match_start,
+					((i == selection) ? PL_STDOUT : 0) |\
```

Comment:
Redundant parens, odd indent, redundant line continuation and redundant spaces below...

Reply:
I dont use spaces, only tabs, in a few cases spaces used to so more clear the one expression down of the other like this one. Try it with tab-width=4... Anyway, it is your code now rewrite it as you wish.